charts/scalar-admin-for-kubernetes/README.md

```diff
@@ -28,5 +28,6 @@ Current chart version is `2.0.0-SNAPSHOT`.
 | scalarAdminForKubernetes.securityContext.runAsNonRoot | bool | `true` | Containers should be run as a non-root user with the minimum required permissions (principle of least privilege). |
 | scalarAdminForKubernetes.serviceAccount.automountServiceAccountToken | bool | `true` | Specify whether to mount a service account token or not. |
 | scalarAdminForKubernetes.serviceAccount.serviceAccountName | string | `""` | Name of the existing service account resource. |
+| scalarAdminForKubernetes.tls.caRootCertSecret | string | `""` | Name of the Secret containing the custom CA root certificate for TLS communication. This chart mounts the root CA certificate file on /tls/certs/ directory. |
```

Comment:
I don't think we need to capitalize `secret` here since it's not a proper noun. But please correct me if I'm wrong!

```suggestion
| scalarAdminForKubernetes.tls.caRootCertSecret | string | `""` | Name of the secret containing the custom CA root certificate for TLS communication. This chart mounts the root CA certificate file on the /tls/certs/ directory. |
```

Reply:
Ah, yes. You are right. It's not a proper noun. I will update it!